### ISSUE TITLE
"template-url" attr doesn't run

### ISSUE THREAD

**Issue by adibalcan** [benmarch/angular-bootstrap-tour#11]
I have a template-url attr on my body (where is tour attribute) and:
```javascript
        function lookupTemplate(templateUrl, scope) {

            var template = $templateCache.get(templateUrl);

            console.log(templateUrl)
            console.log(template)

            if (template) {
                return compileTemplate(template, scope);
            }

            return null;

        }
```

and output is:

```
test.html
undefined
```

**Comment by benmarch** [comment]
My apologies, this is an oversight. This won't make an $http request for the template, it will just look it up in the template cache. I have a quick fix for it, but it will require some testing. I will get it fixed as soon as possible.

**Comment by benmarch** [comment]
Ok this should be resolved now in 0.3.2. Let me know if it still isn't working.
